java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java

```diff
@@ -1028,4 +1036,112 @@ public void testRemoveSpecificAPIpatterns() throws IOException {
             "}")
         .doTest();
   }
+
+  public void transformAndCreateNewPropertyFile(
+      String srcProp, String trgtProp, String stale_flag, boolean isTreated) throws IOException {
+
+    String[] temp = stale_flag.split("_");
+    String methodPropertiesToAdd =
```

Comment:
Can all the strings here go into a custom properties file in the test resources directory?